packages/genesys-spark-components/src/components/stable/gux-dropdown/gux-dropdown.tsx

```diff
@@ -421,13 +422,10 @@ export class GuxDropdown {
 
   private renderOption(option: HTMLGuxOptionElement): JSX.Element {
     let optionText = option.textContent;
-    if (hasSlot(option, 'subtext')) {
-      const subtext = option.querySelector('[slot=subtext]');
-      optionText = optionText.substring(
-        0,
-        optionText.length - subtext.textContent.length
-      );
```

Comment:
I think removing this will change the behavior for existing components with a `subtext` slot but no text slot, which could break deployed apps. So we probably need to preserve this in that specific case, unless I've missed something.

Reply:
Added this back in, and put in a ticket for v5 to review